src/qibolab/runcards/qw5q_gold_qblox.yml

```diff
@@ -326,66 +328,108 @@ native_gates:
                 type: qd                    # qubit drive
             MZ:
                 duration: 2000
-                amplitude: 0.4
-                frequency: 7_118_680_035      # resonator frequency
-                if_frequency: -131_319_965    # difference in resonator frequency
+                amplitude: 0.2
+                frequency: 7_118_627_416      # resonator frequency
+                if_frequency: -136_372_584    # difference in resonator frequency
                 shape: Rectangular()
                 type: ro                    # readout
+
+    two_qubit:
+        3-2:
+            CZ:
+            - duration: 32
+              amplitude: -0.6010
+              shape: Exponential(12, 5000, 0.1)
+              qubit: 3
+              relative_start: 0
+              type: qf
+            - duration: 20
+              amplitude: 0
+              shape: Rectangular())
+              qubit: 3
+              relative_start: 32
+              type: qf
+            - type: virtual_z
+              phase: -3.862
+              qubit: 3
+
+            - duration: 32
+              amplitude: 0
+              shape: Rectangular())
+              qubit: 2
+              relative_start: 0
+              type: qf
+            - duration: 20
+              amplitude: 0
+              shape: Rectangular())
+              qubit: 2
+              relative_start: 32
+              type: qf
+            - type: virtual_z
+              phase: -0.095
+              qubit: 2
```

Comment:
Are all these pulses required? Because it seems like two different ways to do the CZ gate. If yes, we should probably keep only the one that works better in the final runcard that will be merged.